packages/prisma/package.json

```diff
@@ -16,7 +16,8 @@
     "generate-schemas": "prisma generate && prisma format",
     "post-install": "yarn generate-schemas",
     "seed-app-store": "ts-node --transpile-only ./seed-app-store.ts",
-    "delete-app": "ts-node --transpile-only ./delete-app.ts"
+    "delete-app": "ts-node --transpile-only ./delete-app.ts",
+    "seed-analytics": "ts-node --transpile-only ./seed-analytics.ts"
```

Comment:
Running this command we can inject a test user admin account with a team and bookings in the past. We can run it multiple times to add more bookings to DB.